common/src/main/java/io/netty/util/concurrent/FastThreadLocalThread.java

```diff
@@ -70,15 +69,15 @@ public FastThreadLocalThread(ThreadGroup group, Runnable target, String name, lo
      * Returns the internal data structure that keeps the thread-local variables bound to this thread.
      * Note that this method is for internal use only, and thus is subject to change at any time.
      */
-    public final InternalThreadLocalMap threadLocalMap() {
```

Comment:
@normanmaurer , IMHO, May be we use 'protected' access level might be better? That can make subclass able to access this method too.  Same below.